docs/stable/core_extensions/iceberg/iceberg_rest_catalogs.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: docu
+redirect_from: null # maybe redirect from old amazon_s3_tables and amazon_sagemaker_lakehouse docs?
+title: Iceberg Rest Catalogs
+---
+
+The `iceberg` extension supports attaching Iceberg Rest Catalogs. Before attaching an Iceberg Rest Catalog, you must install the `iceberg` extension by following the instructions located in the [overview]({% link docs/stable/core_extensions/iceberg/overview.md %}).
+
+If you are attaching to an Iceberg Rest Catalog managed by Amazon, please see the instructions for attaching to [Amazon S3 tables]({% link docs/stable/core_extensions/iceberg/amazon_s3_tables.md %}) or [Amazon Sagemaker Lakehouse]({% link docs/stable/core_extensions/iceberg/amazon_sagemaker_lakehouse.md %}).
+
+For all other Iceberg Rest Catalogs, you can follow the instructions below. Please see the [Examples](#examples) section for questionsabout specific catalogs.
+
+Most Iceberg Rest Catalogs authenticate via Oauth2. You can use the existing DuckDB secret workflow to create the oauth secret.
```

Comment:
`Oauth2` ->  `OAuth2`
`oauth` -> `oauth2` (or `OAuth2`)

Reply:
Also, "oauth secret", isn't a thing anymore right? We went for `ICEBERG secret` instead